### PR TITLE
1. add Create tests for services (MOV-6)

### PR DESCRIPTION
**Pull request by RomanKashenok**

### CODE REVIEW

File: src/main/java/info/movietrash/cinemabase/dto/UserDto.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.Getter;
```
Comment (RomanKashenok) [inline src/main/java/info/movietrash/cinemabase/dto/UserDto.java:5]: remove

File: src/main/java/info/movietrash/cinemabase/model/Movie.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "name")
+    private String moviename;
+    @Column(name = "poster")
+    private String poster;
+    @Column(name = "year")
+    private String year;
+    @Column(name = "rating_imdb")
+    private Double rating_imdb;
```
Comment (RomanKashenok) [inline src/main/java/info/movietrash/cinemabase/model/Movie.java]: rename please

File: src/main/java/info/movietrash/cinemabase/model/Movie.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    @Column(name = "description")
+    private String description;
+    @Column(name = "age_restrictions")
+    private Integer age_restriction;
```
Comment (RomanKashenok) [inline src/main/java/info/movietrash/cinemabase/model/Movie.java]: rename please

File: src/main/java/info/movietrash/cinemabase/model/Movie.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+@Table(name = "movie")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Movie {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "name")
+    private String moviename;
```
Comment (RomanKashenok) [inline src/main/java/info/movietrash/cinemabase/model/Movie.java]: rename please

File: src/main/resources/sql/create_schema.sql [hunk 5]
```diff
@@ -37,8 +37,8 @@ CREATE TABLE movie
     poster           VARCHAR(50),
     year             VARCHAR(10),
     rating_imdb      DOUBLE    not null,
-    description      MEDIUMTEXT,
-    agere_strictions INT,
+    description      VARCHAR(500),
+    age_restrictions TINYINT(1),
```
Comment (RomanKashenok) [inline src/main/resources/sql/create_schema.sql]: rename to "is_adult"

File: src/test/java/info/movietrash/cinemabase/service/impl/UserServiceImplTest.java [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+    void should_invoke_userRepository_save_when_createUser() {
+        userService.createUser(new UserDto(null, "SSS", "S123456"));
+
+        verify(userRepository).save(any());
+    }
+
+    @Test
+    public void should_returns_users_whentest_findUsers() {
+        List<User> allUsers = new ArrayList<>();
+        allUsers.add(new User(5L, "SSS", "S123456", null, true, null, null));
```
Comment (RomanKashenok) [inline src/test/java/info/movietrash/cinemabase/service/impl/UserServiceImplTest.java:47]: refers to not existing constructor

File: src/main/java/info/movietrash/cinemabase/dto/UserDto.java (hunk 1 above)
Comment (Sergey113222) [reply src/main/java/info/movietrash/cinemabase/dto/UserDto.java:5]: deleted

File: src/main/java/info/movietrash/cinemabase/model/Movie.java (hunk 4 above)
Comment (Sergey113222) [reply src/main/java/info/movietrash/cinemabase/model/Movie.java]: called name

File: src/main/java/info/movietrash/cinemabase/model/Movie.java (hunk 2 above)
Comment (Sergey113222) [reply src/main/java/info/movietrash/cinemabase/model/Movie.java]: ratingImdb

File: src/main/java/info/movietrash/cinemabase/model/Movie.java (hunk 3 above)
Comment (Sergey113222) [reply src/main/java/info/movietrash/cinemabase/model/Movie.java]: private Boolean adult;

File: src/main/resources/sql/create_schema.sql (hunk 5 above)
Comment (Sergey113222) [reply src/main/resources/sql/create_schema.sql]: done

File: src/main/java/info/movietrash/cinemabase/model/Movie.java (hunk 3 above)
Comment (Sergey113222) [reply src/main/java/info/movietrash/cinemabase/model/Movie.java]: @Column(name = "is_adult")

**Review by Sergey113222 — COMMENTED** [review]
fix all

File: src/main/java/info/movietrash/cinemabase/dto/UserDto.java (hunk 1 above)
Comment (RomanKashenok) [reply src/main/java/info/movietrash/cinemabase/dto/UserDto.java:5]: nop. not deleted as I can see

File: src/main/java/info/movietrash/cinemabase/repository/UserRepository.java [hunk 7]
```diff
@@ -1,7 +1,13 @@
 package info.movietrash.cinemabase.repository;
 
+import info.movietrash.cinemabase.model.Role;
 import info.movietrash.cinemabase.model.User;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
```
Comment (RomanKashenok) [inline src/main/java/info/movietrash/cinemabase/repository/UserRepository.java:6]: Remove if not used

File: src/main/java/info/movietrash/cinemabase/repository/UserRepository.java [hunk 8]
```diff
@@ -1,7 +1,13 @@
 package info.movietrash.cinemabase.repository;
 
+import info.movietrash.cinemabase.model.Role;
```
Comment (RomanKashenok) [inline src/main/java/info/movietrash/cinemabase/repository/UserRepository.java:3]: Remove if not used

**Comment by RomanKashenok** [comment]
Also tests not working

**Pull request merged**